src/ui/components/todomvc-app/component.ts

```diff
@@ -74,34 +63,37 @@ export default class TodoMVCApp extends Component {
     }
   }
 
+  setTodos(todos) {
```

Comment:
Could default this `setTodos(todos = this.todos) {` to make it a tad less verbose in the more common cases (non-filtering basically)...